data/112/532/497/7/1125324977.geojson

```diff
@@ -6,6 +6,7 @@
         "EAL"
     ],
     "edtf:cessation":"2017-05-19",
+    "edtf:deprecated":"2023-09-27",
```

Comment:
What's going on with the deprecated localadmin here? Should these be superseded?

Reply:
Eagle eye! Looks like a bug in my process and script, I'll push out a fix. The scope is limited to 13 records across FR, GB, NZ, SE repos so I'll applly manual fixes.

When I selected the feature set of WOF records to work on, I looked for any records that were "mz:is_current" IN (1,-1). But looking closer, this particular record already had a `"edtf:cessation"` set, but `"mz:is_current": 1`. So the fix will be to unset the `"edtf:deprecated"` and toggle `"mz:is_current": 0`.

Reply:
Fixed via https://github.com/whosonfirst-data/whosonfirst-data-admin-gb/pull/90/commits/921fe38fdfc190f1a415a35c43a597386d48e985.